src/exco/util.py

```diff
@@ -9,13 +9,15 @@
 from openpyxl import Workbook
 from openpyxl.cell import Cell
 from openpyxl.utils import get_column_letter
+from openpyxl.worksheet.merge import MergedCellRange
 from openpyxl.worksheet.worksheet import Worksheet
 
 from exco import setting as st
 from exco.cell_full_path import CellFullPath
 
 T = TypeVar('T')
 CellValue = Union[str, int, date, None]
+TupleCellLocation = Tuple[int, int]
```

Comment:
I think there is a coordinate class somewhere please reuse that.

Reply:
Cell coordinate or sth.

Reply:
When I use CellLocation I keep getting a nasty circular import bug because cell_location.py uses util.tuple_to_coordinate (I think).
```
ImportError: cannot import name 'tuple_to_coordinate' from partially initialized module 'exco.util'
```
I have skimmed over the clean code lecture again and tried applying TYPE_CHECKING and wrapped CellLocation in single quotes but it would still fail (either a CellLocation is not defined or circular import error).
However, since the name of the function is get_..._coordinate and the coordinate in CellLocation is a string could I perhaps return a str with type_alias = CellCoordinate?

Reply:
Sorry, the problem is now fixed by importing inside the function. Now the problem is that flake8 complains because the function returns CellLocation before importing CellLocation. Currently working on that!

Reply:
It should work now.